integration_tests/test_zip_source.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2015, 2016 Canonical Ltd
+# Copyright (C) 2015, 2016, 2017 Canonical Ltd
```

Comment:
@sergiusens I've moved the dirs to the setup, with a note that explains how to use them. I think that will make your branch easier, and later I could easily sed all of them if we want to take a different approach.

About the copyright, I've asked Andrew from legal. As soon as he gets back, I'll make sure that all our files comply and are consistent.